client/src/components/ArtDetails.jsx

```diff
@@ -15,19 +18,22 @@ function ArtDetails({ art, onClose }) {
         </button>
       </header>
       <article className="modal-content-artDetails">
-        <img
-          src={art.image}
-          alt={art.title}
-          className="modal-picture-artDetails"
-        />
-
+        {art.image ? (
+          <picture className="modal-picture-artDetails">
```

Comment:
I removed the picture tag because Yoann requested it, so please keep the version that is on dev

Reply:
It's changed 👍